moves/moves.go

```diff
@@ -20,12 +20,17 @@ type Roll interface {
 //grabs the nanosecond as a seed
 func seed() int64 {
 	now := time.Now()
-	return now.Unix()
+	return now.UnixNano()
 }
 
 func (m Move) Roll(bonus int) int {
+	//roll the first d6, +1 since Intn is 0 based
 	rand.Seed(seed())
-	return ((rand.Intn(10)) + 2) + bonus
+	d1 := (rand.Intn(6) + 1)
```

Comment:
That's a lot better actually.  Don't know why I didn't do that the first time.  I'll blame new code environment.